spec/request/api/v1/users/create_spec.rb

```diff
@@ -3,11 +3,17 @@
 require 'rails_helper'
 
 describe 'POST api/v1/users/', type: :request do
+
+  subject do
+    post user_registration_path, params: params, as: :json
+    response
+  end
+
   let(:user)            { User.last }
   let(:failed_response) { 422 }
 
-  describe 'POST create' do
-    let(:email)                 { 'test1@test.com' }
+  context 'POST create' do
```

Comment:
Actually the contexts should be like you did in the other tests, something like `when the params are correct` or `when the params are incorrect`, or something like `when the token has expired`

In this case you used `post create` which sounds more like the text used for the `describe` block